Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM mesosphere/mesos:0.21.1-1.1.ubuntu1404
+
+ENV HDFS_MESOS_VERSION 0.1.0
+
+COPY hdfs-mesos-$HDFS_MESOS_VERSION.tgz /hdfs/
+RUN cd /hdfs && tar xfz hdfs-mesos-$HDFS_MESOS_VERSION.tgz
+
+ENV MESOS_NATIVE_JAVA_LIBRARY /usr/local/lib/libmesos.so
+ENV MESOS_NATIVE_LIBRARY /usr/local/lib/libmesos.so
+
+WORKDIR /hdfs/hdfs-mesos-$HDFS_MESOS_VERSION
+
+CMD ./bin/hdfs-mesos
```

Comment:
I agree. If we were reading the config file into a org.apache.hadoop.conf.Configuration object and then rendering that, we would be able to override specific config values at the start of the framework with system properties i.e. passing `JAVA_OPTS="-Dmesos.master.uri=zk://somehost/mesos"` as an environmental variable to the start of the framework. This can be tackled in a separate PR.